libcob/common.h

```diff
@@ -989,14 +991,20 @@ enum cob_open_mode {
 #define COB_READ_MASK		\
 	(COB_READ_NEXT | COB_READ_PREVIOUS | COB_READ_FIRST | COB_READ_LAST)
 
-/* I-O status */
+/* I-O status - TODO: these should have internal only values; and then
+   map later to an i-o status "per dialect", inluding alphanumeric 0x
+   and 9/123 status values,
+   will be move to fileio.h in 4.x on remove of OC extfh */
```

Comment:
Can we already move that to fileio.h now?
If not, then please note the reason in the new TODO entry.

Reply:
Not sure, that would require changes in `common.c` too (`cob_fatal_error` uses these status codes too).

Reply:
Added to TODO-list/